agent/tcs/handler/handler.go

```diff
@@ -112,7 +118,7 @@ func startSession(url string, cfg *config.Config, credentialProvider *credential
 	timer := time.AfterFunc(utils.AddJitter(heartbeatTimeout, heartbeatJitter), func() {
```

Comment:
Can you do the same as acs to move the timer after the connection was established?

Reply:
Yeah, I thought I did that. But, obviously didn't. I'll modify this.